openpower/package/hostboot/hostboot-0005-pcie-swap-fix.patch

```diff
@@ -0,0 +1,62 @@
+From dcd978f55a2cdf8c5dbce11a35b484de3e81fb6a Mon Sep 17 00:00:00 2001
+From: Norman James <nkskjames@gmail.com>
+Date: Tue, 21 Apr 2015 15:24:41 -0500
+Subject: [PATCH] fix pcie lane swap
+
```

Comment:
Please upload this to Gerrit and make sure the patch's Change-Id matches the Gerrit change ID.  We have some toolling coming to automatically remove patches when the real commit surfaces.